src/components/exchange/ConfirmExchangeButton.js

```diff
@@ -99,6 +99,23 @@ export default function ConfirmExchangeButton({
     };
   }, [asset.address, colorForAsset, colors, isDarkMode, isSwapDetailsRoute]);
 
+  /* leaving this here so u know the 🔑's
+    let label =
+    type === ExchangeModalTypes.deposit
+      ? lang.t('button.confirm_exchange.deposit')
+      : type === ExchangeModalTypes.withdrawal
+      ? lang.t('button.confirm_exchange.withdraw')
+      : lang.t('button.confirm_exchange.swap');
+  if (!isSufficientBalance) {
+    label = lang.t('button.confirm_exchange.insuffecient_funds');
+  } else if (!isSufficientGas) {
+    label = lang.t('button.confirm_exchange.insuffecient_eth');
+  } else if (slippage > SlippageWarningTheshold) {
+    label = lang.t('button.confirm_exchange.swap_anyway');
+  } else if (disabled) {
+    label = lang.t('button.confirm_exchange.enter_amount');
+    */
+
```

Comment:
@skylarbarrera Is there a reason to keep this as a comment, as opposed to just using it as the source code for the section below? I can change it over and make sure the translations match, if that makes sense. Thanks!

Reply:
the logic looked a little different so while i was catching up the branch i left for you to use for reference, those translations should be good, i think we're most likely just missing a few states